compiler/runtime/Helpers.inc

```diff
@@ -353,6 +353,8 @@ SETVAL(TR_PPCAESDecrypt, TR_FSRH+86)
 SETVAL(TR_PPCVectorLogDouble, TR_FSRH+87)
 SETVAL(TR_PPCquadWordArrayCopy_vsx, TR_FSRH+88)
 SETVAL(TR_PPCforwardQuadWordArrayCopy_vsx, TR_FSRH+89)
+SETVAL(TR_PPCPostP10forwardArrayCopy, TR_FSRH+90)
+SETVAL(TR_PPCPostP10genericArrayCopy, TR_FSRH+91)
```

Comment:
The numbering for the rest of the helpers in this table needs to be updated. As it stands, these new helpers are indistinguishable from `TR_PPCLEArrayCopy` and `TR_PPCforwardLEArrayCopy`.

Reply:
That is intentional for the time being, before openj9 side is checked in.